### PR TITLE
Add candidate summary and fix amplify setup step 1

### PR DESCRIPTION
**Pull request by one0212**
1. Add candidate summary
2. Fix amplify setup step 1

### CODE REVIEW

File: src/containers/CandidateListPage/TestList/index.js [hunk 1]
```diff
@@ -3,15 +3,21 @@ import PropTypes from 'prop-types';
...
 
+import CandidateSummary from './CandidateSummary/index';
+
 class TestList extends React.Component {
   state = {
     delConfirmModalVisible: false,
     delTest: null,
     delAnime: false,
+    testResultModalVisible: false,
+    testRedultModalTarget: [],
```
Comment (HY-Ku) [inline src/containers/CandidateListPage/TestList/index.js]: typo: Redult -> Result

File: src/containers/CandidateListPage/TestList/index.js [hunk 2]
```diff
@@ -40,6 +46,23 @@ class TestList extends React.Component {
...
+    this.setState({
+      testResultModalVisible: true,
+      testRedultModalTarget: [
+        e.target.getAttribute('candidate'),
+        e.target.getAttribute('interviewer'),
+      ],
+    });
+  };
+
+  testRedultModalCancel = () => {
```
Comment (HY-Ku) [inline src/containers/CandidateListPage/TestList/index.js]: typo: Redult -> Result

File: src/containers/CandidateListPage/TestList/index.js [hunk 3]
```diff
@@ -40,6 +46,23 @@ class TestList extends React.Component {
     }, 500);
   };
 
+  showTestResultModal = e => {
+    // console.log(e.target.getAttribute('candidateName'));
+    this.setState({
+      testResultModalVisible: true,
+      testRedultModalTarget: [
```
Comment (HY-Ku) [inline src/containers/CandidateListPage/TestList/index.js]: typo: Redult -> Result

**Review by HY-Ku — CHANGES_REQUESTED** [review]

File: src/containers/CandidateListPage/TestList/index.js (hunk 2 above)
Comment (one0212) [reply src/containers/CandidateListPage/TestList/index.js]: done

File: src/containers/CandidateListPage/TestList/index.js (hunk 1 above)
Comment (one0212) [reply src/containers/CandidateListPage/TestList/index.js]: done

**Review by HY-Ku — APPROVED** [review]

**Pull request merged**